internal/datacoord/segment_manager.go

```diff
@@ -439,8 +442,7 @@ func (s *SegmentManager) SealAllSegments(ctx context.Context, collectionID Uniqu
 			ret = append(ret, id)
 			continue
 		}
-		if info.State == commonpb.SegmentState_Flushing ||
-			info.State == commonpb.SegmentState_Flushed {
+		if info.State != commonpb.SegmentState_Growing {
```

Comment:
The current design is that we manually altering a segment's state from `Importing` to `Sealed`, so bulk loading segment can be picked up by flush procedure just like other segments. This is done by calling `Flush` after task enters a `Persisted` state:
https://github.com/milvus-io/milvus/blob/master/internal/rootcoord/root_coord.go#L1806

So might wanna add Importing here as well.

Otherwise we should modify flush policy to catch `Importing` segments here:
https://github.com/milvus-io/milvus/blob/master/internal/datacoord/segment_allocation_policy.go#L186